napari/_qt/tests/test_qt_viewer.py

```diff
@@ -20,6 +20,7 @@ def test_qt_viewer(qtbot):
     assert viewer.dims.ndim == 2
     assert view.dims.nsliders == viewer.dims.ndim
     assert np.sum(view.dims._displayed_sliders) == 0
+    view.shutdown()
```

Comment:
Hmmm, it’s mildly annoying to have these in every test. Is there a way to use the view fixture to have the teardown after the yield statement happen automatically?

Reply:
If we can make the syntax nice enough, we might even be able to start testing our examples again! 🎉

Reply:
Oh yea these test actually all getting deleted in #723 so theres only one view.shutdown() in the parametrized test. I used the fixture yeild teardown model in `test_qt_play.py` so that should work for the examples!